### PR TITLE
Fixes to proxy and failing script.

### PR DESCRIPTION
**Pull request by Mionsz**
Fixes to proxy and failing script.

### CODE REVIEW

File: scripts/setup_build_env.sh [hunk 1]
```diff
@@ -82,7 +82,7 @@ function get_and_patch_intel_drivers()
         error "MTL patch for ICE=v${ICE_VER} could not be found: ${MTL_DIR}/patches/ice_drv/${ICE_VER}"
         return 1
     fi
-    wget_download_strip_unpack "https://downloadmirror.intel.com/${IRDMA_DMID}/irdma-${IRDMA_VER}.tgz" "${IRDMA_DIR}"
+    NO_PROXY="" no_proxy="" wget_download_strip_unpack "https://downloadmirror.intel.com/${IRDMA_DMID}/irdma-${IRDMA_VER}.tgz" "${IRDMA_DIR}"
```
Comment (Sakoram) [inline scripts/setup_build_env.sh:85]: 1. Why `no_proxy` added to the script? I use it as well and I pass it to the script itself, but I think some users would like to use their own proxy setup.
2. Please move installation of irdma after the ice driver installation. (to line 88)

File: scripts/common.sh [hunk 2]
```diff
@@ -258,31 +255,70 @@ function print_logo_anim()
...
     prompt "Configuration of iRDMA starting."
-    prompt "Enabling RoCE."
+    touch "/etc/modprobe.d/irdma.conf"
 
-    # enable RoCE
-    roce_ena_val=$(grep "options irdma roce_ena=" /etc/modprobe.d/irdma.conf | cut -d "=" -f 2)
-    if [[ -z "$roce_ena_val" ]]; then
+    prompt "Enabling RoCE."
+    if grep -e "^\s*options\s\+irdma\s\+roce_ena=" /etc/modprobe.d/irdma.conf 1>/dev/null 2>&1; then
+        sudo sed -i "s/^\s*options\s\+irdma\s\+roce_ena=\(0\|1\)/options\ irdma\ roce_ena=1/g" /etc/modprobe.d/irdma.conf
```
Comment (Sakoram) [inline scripts/common.sh:308]: I guess it gets the job done, but God, it's so hard to read

**Comment by Sakoram** [comment]
We need a few changes to the script:
1. Move irdma installation after ice installation.
2. Move libfabric installation after irdma installation

**Comment by Mionsz** [comment]
> We need a few changes to the script:
> 
> 1. Move irdma installation after ice installation.
> 2. Move libfabric installation after irdma installation

Ok, I've fixed the order in the last commit. Of course this was done in the setup_build_env.sh

File: scripts/setup_build_env.sh [hunk 3]
```diff
@@ -89,22 +54,25 @@ function get_and_patch_intel_drivers()
...
+    set +x
 }
 
 function build_install_and_config_intel_drivers()
 {
+    set -x
+    make -j "${NPROC}" -C "${IAVF_DIR}/src" install
+    make -j "${NPROC}" -C "${ICE_DIR}/src" install
     pushd "${IRDMA_DIR}"
     ./build.sh
```
[the remark this answers is not in the document]
Comment (Sakoram) [reply scripts/setup_build_env.sh:50]: ?

[the remark this answers is not in the document]
Comment (Mionsz) [reply scripts/setup_build_env.sh:50]: added

**Review by Sakoram — APPROVED** [review]
Looks good

**Pull request merged**